OpenSim/Simulation/Model/PhysicalFrame.h

```diff
@@ -140,17 +136,16 @@ class OSIMSIMULATION_API PhysicalFrame : public Frame {
         ScaleFactors */
     void scale(const SimTK::Vec3& aScaleFactors);
 
-    /** @name DEPRECATED API 
+    /** Add a Mesh specified by file name to the list of Geometry owned by the ModelComponent.
+    Scale defaults to 1.0 but can be changed on the call line.
+    */
+    void addMeshGeometry(const std::string &aGeometryFileName, const SimTK::Vec3 scale = SimTK::Vec3(1));
```

Comment:
Why the special case for adding Mesh geometry and why only on PhysicalFrame? If MeshGeometry is Geometry, then `ModelComponent::addGeometery()` should be sufficient. Why not create the Geometry first then add it like any other Geometry?